build.sbt

```diff
@@ -50,7 +50,9 @@ lazy val commonSettings = Seq(
           <url>http://haifengl.github.io/</url>
         </developer>
       </developers>
-  )
+  ),
+  EclipseKeys.withSource := true,
```

Comment:
What do these two lines mean?

Reply:
https://github.com/sbt/sbteclipse/wiki/Using-sbteclipse#withsource
https://github.com/sbt/sbteclipse/wiki/Using-sbteclipse#withjavadoc
basically, these two allows to download source and java doc for convenience reference

Reply:
https://github.com/sbt/sbteclipse/wiki/Using-sbteclipse#withsource
https://github.com/sbt/sbteclipse/wiki/Using-sbteclipse#withjavadoc
basically, these two allows to download source and java doc for convenience reference